internal/ecrconsumer/sqs.go

```diff
@@ -0,0 +1,86 @@
+package ecrconsumer
+
+import (
+	"context"
+	"time"
+
+	"github.com/Wattpad/sqsconsumer"
+	"github.com/Wattpad/sqsconsumer/middleware"
+
+	// Metric Imports
+	kitlog "github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/metrics"
+	"github.com/go-kit/kit/metrics/dogstatsd"
+)
+
+// NOTE: Refactor to take Env struct
+
+type SQSConfig struct {
```

Comment:
This config type can be removed, the consumer only depends on the queue name which can be passed as a parameter.